docs/Developers.md

```diff
@@ -6,6 +6,7 @@
   * [Building SRT](#building-srt)
 * [Project Structure](#project-structure)
 * [Coding Rules](#rules)
```

Comment:
```suggestion
* [Coding Rules](#coding-rules)
```

Reply:
That was yours! :D 

Ok, I'll fix the target stub section as well.